src/styles/book.css

```diff
@@ -0,0 +1,125 @@
+.d-flex {
+  display: flex;
+}
+
+.book-container {
+  font-family: 'Roboto Slab', serif;
+  justify-content: space-between;
+  align-items: center;
+  border: solid 1px rgb(214, 214, 214);
+  border-radius: 4px;
+  margin-left: -41px;
+  margin-top: 10px;
+  height: 7rem;
+  padding: 10px 5%;
+  background-color: #fff;
+}
+
+.general-info {
+  flex-flow: column wrap;
+  width: 30%;
+}
+
+.title {
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0;
+}
+
+.book-management {
+  list-style-type: none;
+  padding: 0;
+}
+
+.current-chapter > button {
+  width: 200px;
```

Comment:
- [ ] Kindly fix your design to match the zipline design, you can fix it by removing `width:200px;` 
> Your design 
![image](https://user-images.githubusercontent.com/53402600/208126178-e1ca8f2a-85b4-4821-8d98-8b344f427b96.png)